### PR TITLE
Clean up CSS and HTML formatting

### PR DESCRIPTION
**Pull request by jazeee**
Replace tabs with 2 spaces, and then do an initial pass of formatting
in the HTML.
(We'd probably benefit from moving the code out of HTML
and into a lintable file.)

Add README to help with formatting

### CODE REVIEW

File: ui/roll20_character_sheet_0.2.css [hunk 1]
```diff
@@ -1,953 +1,1080 @@
...
+  text-align-last: center;
 }
 
 /* Change the character for dice roll buttons to be
    the two joined squares character and give them the blue background */
 button.btn.ui-draggable[type=roll] {
-    background-image: linear-gradient(90deg, #ddf, #99f);
+  background-image: linear-gradient(90deg, #ddf, #99f);
 }
+
```
Comment (jazeee) [inline ui/roll20_character_sheet_0.2.css:34]: In VSCode, I ran the auto-formatter, which made these files more consistent.
Consider reviewing ignoring whitespace:
https://github.com/epic-power-rpg/roll20/pull/10/files?diff=unified&w=1

**Comment by jazeee** [comment]
Note - if you are in the midst of making code changes, I can always delete this PR and re-do the auto-formatting.

File: ui/roll20_character_sheet_0.2.css [hunk 2]
```diff
@@ -1,953 +1,1080 @@
...
-    height: 24px;
-}    
+  height: 24px;
+}
 
 span.sheet-attribute-value {
-    display: inline-block; /* Make the height work for span. */
-    border-color: transparent;
+  display: inline-block;
+  /* Make the height work for span. */
```
Comment (johnlamping) [inline ui/roll20_character_sheet_0.2.css:254]: This comment applies to the line before, so it should move up a line.

File: ui/roll20_character_sheet_0.2.css [hunk 3]
```diff
@@ -1,953 +1,1080 @@
...
 }
 
 /* ---------- Roll section ------------- */
 
 .sheet-roll-section {
-    display: flex;
-    margin-top: -26px;  /* Get rid of the space the sheet puts above us. */
+  display: flex;
+  margin-top: -26px;
+  /* Get rid of the space the sheet puts above us. */
```
Comment (johnlamping) [inline ui/roll20_character_sheet_0.2.css:320]: Same here: Needs to move up a line.

File: ui/roll20_character_sheet_0.2.css [hunk 4]
```diff
@@ -1,953 +1,1080 @@
...
-    flex-direction: row;
-    align-items: center;
-    margin-top: -1px; /* and make rows overlap */
+
+.sheet-skills div[data-groupname]>div {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: -1px;
+  /* and make rows overlap */
```
Comment (johnlamping) [inline ui/roll20_character_sheet_0.2.css:813]: Same here: Needs to move up a line.

File: ui/roll20_character_sheet_0.2.css [hunk 5]
```diff
@@ -1,953 +1,1080 @@
...
-    flex-direction: row;
-    align-items: center;
-    margin-top: -1px; /* and make rows overlap */
+
+.sheet-equipment div[data-groupname]>div {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: -1px;
+  /* and make rows overlap */
```
Comment (johnlamping) [inline ui/roll20_character_sheet_0.2.css:924]: Same here: Needs to move up a line.

File: ui/roll20_character_sheet_0.2.css [hunk 6]
```diff
@@ -1,953 +1,1080 @@
...
-    flex-direction: column;
-    align-items: stretch;
-    margin-top: -1px; /* and make rows overlap */
+
+.sheet-spells div[data-groupname]>div {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  margin-top: -1px;
+  /* and make rows overlap */
```
Comment (johnlamping) [inline ui/roll20_character_sheet_0.2.css:1050]: Same here: Needs to move up a line.

File: ui/roll20_character_sheet_0.2.css [hunk 7]
```diff
@@ -1,953 +1,1080 @@
...
 
-.sheet-spell-row > input ~ input,
-.sheet-spell-row > input ~ select,
-.sheet-spell-row > select ~ input {
-    margin-left: -1px; /* Make consecutive buttons overlap */
+.sheet-spell-row>input~input,
+.sheet-spell-row>input~select,
+.sheet-spell-row>select~input {
+  margin-left: -1px;
+  /* Make consecutive buttons overlap */
```
Comment (johnlamping) [inline ui/roll20_character_sheet_0.2.css:1063]: Same here: Needs to move up a line.

File: ui/roll20_character_sheet_0.2.css [hunk 8]
```diff
@@ -1,953 +1,1080 @@
...
+  text-align: right
 }
+
+.sheet-notenote {
+  width: 500px;
+}
+
+.sheet-notes div[data-groupname]>div {
+  margin-top: -1px;
+  /* and make rows overlap */
```
Comment (johnlamping) [inline ui/roll20_character_sheet_0.2.css:1078]: Same here: Needs to move up a line.

**Review by johnlamping — CHANGES_REQUESTED** [review]
There were a few places where the linter moved a comment after the line it applies do. Other than that, looks good.

P.S. This code is extremely ugly, which has made it hard to debug, and has led to it becoming even uglier as I added to it. I don't have a good sense of how to write good css, but this is certainly not it. If you have any suggestions on how to improve it, I'd love to hear them.

**Comment by jazeee** [comment]
> There were a few places where the linter moved a comment after the line it applies do. Other than that, looks good.
> 
> P.S. This code is extremely ugly, which has made it hard to debug, and has led to it becoming even uglier as I added to it. I don't have a good sense of how to write good css, but this is certainly not it. If you have any suggestions on how to improve it, I'd love to hear them.

I'll be digging further into how to clean things up. Auto-formatting gets us started, but I'm going to need to dig in further on it.

**Review by johnlamping — APPROVED** [review]
Thanks!

**Pull request merged**